A45-retry-stats.md

```diff
@@ -58,6 +58,15 @@ We add the following per-overall-client-call metrics to OpenCensus:
 | grpc.io/client/retry\_delay\_per\_call         | retry\_delay\_per\_call         | distribution | grpc\_client\_method |
 
 
+### Tracing
+
+If retry is not enabled on the client channel, the gRPC tracing module should behave the same as it does today. If retry is enabled, the gRPC tracing module should create a tracing span for each call from the perspective of the application, and create a child span for each individual call attempt, including transparent retry attempts. The parent span is named in the form of `Sent.<service name>.<method name>`, which is the same as the case when retry is not enabled. The span for each individual call attempt is named in the form of `Attempt.<service name>.<method name>`, and is with the following additional span attributes:
+
+      `previous-rpc-attempts` : number of preceding attempts,
+      `transparent-retry`     : whether the attempt is a transparent retry.
```

Comment:
What is the format of the value here?  Is it integer 0 or 1, boolean, or string?

Reply:
Specified.